tests/models/test_implicit_als.py

```diff
@@ -71,7 +80,7 @@ def dataset(self) -> Dataset:
             ),
         ),
     )
-    @pytest.mark.parametrize("fit_features_together", (True, False))
+    @pytest.mark.parametrize("fit_features_together", (False,))  # return True option after
```

Comment:
Can you please give a bit more detailed comment?
1. It's not clear what `return` means when you are just reading the code without knowing its history
2. After what?

Reply:
Fixed